lib/test/cdo/crowdin/test_legacy_utils.rb

```diff
@@ -80,40 +79,30 @@ def setup
   end
 
   def test_downloading_is_idempotent
-    # +download_changed_files+ should return the same results (files_to_sync_out_json)
```

Comment:
Rather than removing this comment (and the ones similarly removed below), what about just removing the reference to the removed file, but keeping the rest of the comment in place? Seems to me like the comment still has value even for the etags-only version of the test